### PR TITLE
don't initialise django app within tasks.py

### PR DESCRIPTION
**Pull request by aahunter**
Recently added tasks.py was triggering django initialisation, which is not what we want in prod. Initialisation of Django needs to take place in the web app.

### CODE REVIEW

File: rdrf/tasks.py [hunk 1]
```diff
@@ -1,32 +1,31 @@
 """
 Define events for use with the uWSGI cron-like interface (https://uwsgi-docs.readthedocs.io/en/latest/Cron.html)
 
+Care must be taken with Django imports. We want application initiliasation to be triggered by the web application
```
Comment (sztamas) [inline rdrf/tasks.py]: Spelling error: initialisation

File: rdrf/tasks.py [hunk 2]
```diff
@@ -1,32 +1,31 @@
...
-    def debug_timer(num):
-        logger.debug("")
-
-
 @timer(300)
 def django_cron_heartbeat(num):
     """
     A heartbeat for django-cron (https://github.com/Tivix/django-cron)
     """
+    if apps.apps_ready is not True:
```
Comment (sztamas) [inline rdrf/tasks.py]: `if not apps.apps_ready:`

File: rdrf/tasks.py [hunk 3]
```diff
@@ -1,32 +1,31 @@
...
+        # Django is not initialised, the logger config in settings will not be active
+        print("Deferring calling django-cron heartbeat until django app is ready")
```
Comment (sztamas) [inline rdrf/tasks.py]: Why not use logging instead of `print`?

Comment (aahunter) [inline rdrf/tasks.py]: Because the app is not initialised and therefore there is no logging config.

**Pull request merged**